GitUI/CommandsDialogs/RepoHosting/CreatePullRequestForm.cs

```diff
@@ -132,31 +141,38 @@ private void PopulateBranchesComboAndEnableCreateButton(IHostedRemote remote, Co
                     {
                         await TaskScheduler.Default;
 
-                        IHostedRepository hostedRepository = remote.GetHostedRepository();
-                        var branches = hostedRepository.GetBranches();
+                        try
+                        {
+                            IHostedRepository hostedRepository = remote.GetHostedRepository();
+                            var branches = hostedRepository.GetBranches();
 
-                        await this.SwitchToMainThreadAsync();
+                            await this.SwitchToMainThreadAsync();
 
-                        comboBox.Items.Clear();
+                            comboBox.Items.Clear();
 
-                        var selectItem = 0;
-                        var defaultBranch = hostedRepository.GetDefaultBranch();
-                        for (var i = 0; i < branches.Count; i++)
-                        {
-                            if (branches[i].Name == defaultBranch)
+                            var selectItem = 0;
+                            var defaultBranch = hostedRepository.GetDefaultBranch();
+                            for (var i = 0; i < branches.Count; i++)
                             {
-                                selectItem = i;
+                                if (branches[i].Name == defaultBranch)
+                                {
+                                    selectItem = i;
+                                }
+
+                                comboBox.Items.Add(branches[i].Name);
                             }
 
-                            comboBox.Items.Add(branches[i].Name);
-                        }
+                            if (branches.Count > 0)
+                            {
+                                comboBox.SelectedIndex = selectItem;
+                            }
 
-                        if (branches.Count > 0)
+                            _createBtn.Enabled = true;
+                        }
+                        catch (Exception ex)
                         {
-                            comboBox.SelectedIndex = selectItem;
+                            MessageBox.Show(string.Format(_strRemoteInError.Text, remote.DisplayData) + Environment.NewLine + Environment.NewLine + ex.Message, TranslatedStrings.Error, MessageBoxButtons.OK, MessageBoxIcon.Warning);
```

Comment:
A better error presentation:

```
[Error]

Remote: <xyz/uvw>
Error: Failed to load branches.
<...>
```

```
[Error]

Remote: <xyz/uvw>
Error: Remote ignored.
<...>
```

Reply:
@RussKie Is that what you have in mind?

![image](https://user-images.githubusercontent.com/460196/117469195-bd6c3b80-af55-11eb-92c3-1cfa8e2baff2.png)

![image](https://user-images.githubusercontent.com/460196/117469302-db39a080-af55-11eb-8bf6-12fccec89502.png)

Reply:
Yes, though it looks a little awkward. 

I think it may look better if tweak it a little more:

* use TaskDialog (preferably from System.Windows.Forms namespace)
* Reformat as:
```
[ Remote ignored ] (<-- caption)
---------------------
The GitHub authentica..... (<-- main instruction aka Heading)

Remote: xx/yy/zz (Text)
```

Do you think you could try that?

Reply:
I did something like that using `Microsoft.WindowsAPICodePack.Dialogs`.
Indeed it was looking better.
I rebased and covert it to `System.Windows.Forms.TaskDialog`.
Unfortunately, the plugins no more load in .net 5 version.

So even if it compiles, we are unable to test it and we will have to wait for plugins loading....